packages/components/segmented-control/tests/segmented-control.test.tsx

```diff
@@ -77,4 +77,66 @@ describe("<SegmentedControl />", () => {
       },
     )
   })
+
+  test("focus moves correctly between buttons", async () => {
+    const { user } = render(
+      <SegmentedControl>
+        <SegmentedControlButton value="one" data-testid="firstButton">
+          One
+        </SegmentedControlButton>
+        <SegmentedControlButton value="two" data-testid="secondButton" disabled>
+          Two
+        </SegmentedControlButton>
+        <SegmentedControlButton value="three" data-testid="thirdButton">
+          Three
+        </SegmentedControlButton>
+      </SegmentedControl>,
+    )
+    const firstButton = screen.getByTestId("firstButton")
+    const secondButton = screen.getByTestId("secondButton")
+    const thirdButton = screen.getByTestId("thirdButton")
```

Comment:
This is better.
```tsx
    const firstButton = screen.getByRole("radio", { name: /one/i })
    const secondButton = screen.getByRole("radio", { name: /two/i })
    const thirdButton = screen.getByRole("radio", { name: /three/i })
```

Reply:
@cidkumagai 
That is strange, but I don't think we need to worry too much about it. (Since it only changes a little.)
The test coverage figures are only an indicator; it is more important that the implemented functionality is tested correctly.

Reply:
@108yen 
Thank you very much.
I will make the corrections.